test/test_db/test_db.py

```diff
@@ -80,6 +80,10 @@ def test_trigger_delete_children(self):
         # Assert test child record has been deleted
         self.assertTrue(row is None)
 
+    def test_trigger_delete_role(self):
+        """Unit tests for trigger function delete_role."""
+        pass
```

Comment:
Are the empty unit tests in this file still going to be updated in this PR?

Reply:
Yes I was planning to fill them, I added them as placeholders to remember. I will add TODO.